app/containers/User.js

```diff
@@ -25,7 +27,7 @@ export default class MainPage extends Component {
       console.log(error);
     });
 
-    axios.get("https://make-it-happen-api.herokuapp.com/api/day/count")
+    axios.get("https://make-it-happen-api.herokuapp.com/api/days/count")
```

Comment:
axios.get("https://make-it-happen-api.herokuapp.com/api/days/count", {
        access_token: this.props.accessToken })